lib/s3routes/routes.js

```diff
@@ -134,9 +136,10 @@ function checkTypes(req, res, params, logger) {
  *  values for whether queries are supported
  * @param {function} params.dataRetrievalFn - function to retrieve data
  * @param {RequestLogger} logger - werelogs logger instance
+ * @param {String} [s3config] - s3 configuration
  * @returns {undefined}
  */
-function routes(req, res, params, logger) {
+function routes(req, res, params, logger, s3config) {
     checkTypes(req, res, params, logger);
```

Comment:
Should you add the new value to the check types method?

Reply:
It is optional, I will add type check if it is required.

Reply:
I added a check now.